incubator/nfs/templates/nfs-busybox-rc.yaml

```diff
@@ -0,0 +1,32 @@
+# This mounts the nfs volume claim into /mnt and continuously
+# overwrites /mnt/index.html with the time and hostname of the pod.
+
+apiVersion: v1
+kind: ReplicationController
```

Comment:
ReplicationController? Really? You'll probably want to use a Deployment.

Reply:
I was trying to think of a justification for this @unguiculus because the ReplicationController is also still used in kubernetes/examples version of this repo, and I'm assuming it should be updated to use a Deployment there too, but it still hasn't... https://github.com/kubernetes/examples/tree/master/staging/volumes/nfs

Then I realized that the Deployment resource kind itself still is not v1 API, so presumably isn't being used in any examples yet.  Actually when I realized this is when I started looking into the possibility of using Terraform and dependency ordering to get the Service cluster IP into the PV spec (template).

The Terraform k8s provider has stated explicitly that they will not be supporting Deployment or any other beta or alpha APIs until they are marked v1, most likely because of the risk that would pose for people making production instances and collecting `tfstate` via production uses of Terraform.

So if I go the route of using Terraform to connect PVs to the NFS service because the code in K8S that connects PVs to PVCs still uses only host DNS instead of cluster DNS (and not all hosts outside of Google COS will be using cluster DNS on the host)... we will still need to have a ReplicationController backing the pods for those services for now.